config/settings/base.py

```diff
@@ -283,8 +283,12 @@
 # By default, a version of the model's verbose_name is used (with " page"
 # removed from the end).
 PUBLIC_PAGE_TYPE_LABEL_OVERRIDES = {
-    "articles.articlepage": "Story",
+    "articles.articlepage": "The story of",
+    "articles.focusedarticlepage": "Focus on",
     "articles.recordarticlepage": "Record revealed",
+    "collections.topicexplorerpage": "Explore by topic",
+    "collections.timeperiodexplorerpage": "Explore by time period",
+    "collections.highlightgallerypage": "In pictures",
```

Comment:
SUGGESTION: On this bit, could we not use the `title_label` that the pages have? We could add this to the other page types (e.g. topic explorer, highlight gallery, etc). Story of, Focus on, and Record revealed are already on these pages. Might make things easier?

Reply:
We would need to make these title case of course, which I meant to do at some point anyways - currently they're all caps in the BE, but really we should be capitalising them in the FE :)

Reply:
Okay, leave it with me :)

Reply:
Okay, I've managed to come up with something that works. I had to come up with a way to only show the labels in places when desired (for example, we don't want the 'Explore by topic' label to output for every item on the topic list page), hence why it took so long.